daemons/fenced/fenced_cib.c

```diff
@@ -591,7 +591,7 @@ setup_cib(void)
 
     do {
         sleep(retries);
-        rc = cib_api->cmds->signon(cib_api, CRM_SYSTEM_STONITHD, cib_command);
+        rc = cib_api->cmds->signon(cib_api, crm_system_name, cib_command);
```

Comment:
Yeah I cheated a little bit :) That gets taken care of by later commits, when `crm_is_daemon_name()` gets replaced by `pcmk__parse_server()`. The only place it matters is where the CIB checks whether an IPC client is a daemon, and since those are always local connections, we don't have to worry about compatibility issues.